manifests/profile/www_lib/vhosts/staff_lib.pp

```diff
@@ -0,0 +1,168 @@
+        path           => "${vhost_root}/cgi",
+        allow_override => ['None'],
+        options        => ['None'],
+        require        => $nebula::profile::www_lib::apache::default_access,
+      },
+      {
+        provider       => 'directory',
+        path           => "${vhost_root}/alida-tmp/current/public",
+        allow_override => ['None'],
+        options        => ['FollowSymlinks'],
+        require        => $nebula::profile::www_lib::apache::default_access,
+      },
+      {
+        # Deny access to raw php sources by default
+        # To re-enable it's recommended to enable access to the files
+        # only in specific virtual host or directory
+        provider => 'filesmatch',
+        path     => '.+\.phps$',
+        require  => 'all denied'
+      },
+      {
+        # Deny access to files without filename (e.g. '.php')
+        provider => 'filesmatch',
+        path     => '^\.ph(ar|p|ps|tml)$',
+        require  => 'all denied'
+      },
+    ] + $php5_dirs,
+
+    # Don't allow passive auth for directories still protected by auth system
+    cosign_public_access_off_dirs => [
+      # results in odd looping behavior
+      # {
+      #   provider => 'location',
+      #   path     => '/user/login'
+      # },
+      {
+        provider => 'directory',
+        path     => "${docroot}/funds_transfer",
+      },
+      {
+        provider => 'directory',
+        path     => "${docroot}/sites/staff.lib.umich.edu.funds_transfer",
+      },
+      {
+        provider => 'directory',
+        path     => "${docroot}/linkscan",
+      },
+      {
+        provider => 'directory',
+        path     => "${docroot}/linkscan117",
+      },
+      {
+        provider => 'directory',
+        path     => "${docroot}/pagerate",
+      },
+      {
+        provider => 'directory',
+        path     => "${docroot}/ts",
+      },
+      {
+        provider => 'location',
+        path     => '/alida',
+      },
+    ],
+
+    request_headers               => [
+      # Setting remote user for 2.4
+      'set X-Remote-User "expr=%{REMOTE_USER}"',
+      # Fix redirects being sent to non ssl url (https -> http)
+      'set X-Forwarded-Proto "https"',
+      # Remove existing X-Forwarded-For headers; mod_proxy will automatically add the correct one.
+      'unset X-Forwarded-For',
+    ],
+
+    rewrites                      => [
+      {
+        comment      => 'Serve static assets, retire June 2021',
+        rewrite_cond => '/www/staff.lib/alida-tmp/app/current/public/$1 -f',
+        rewrite_rule => '^/alida/(.*)$  /www/staff.lib/alida-tmp/app/current/public/$1 [L]'
+      },
+      {
+        rewrite_rule => '^(/alida.*)$ http://app-alida:40160$1 [P]'
+      },
+    ],
+
+    custom_fragment               => @(EOT)
+      ProxyPassReverse /alida http://app-alida:40160/
+
+      # Enable http authorization headers
+      <IfModule setenvif_module>
+        SetEnvIfNoCase ^Authorization$ "(.+)" HTTP_AUTHORIZATION=$1
```

Comment:
We don't need to include the <IfModule> there. We should just be able to do

```
setenvifnocase => [ '^Authorization$ "(.+)" HTTP_AUTHORIZATION=$1' ]
``

Reply:
I've added this to the www_lib_vhost profile.